content/Default/apps/content.json

```diff
@@ -1282,11 +1282,7 @@
         "com.endlessm.mayan_languages-screenshot2.jpg",
         "com.endlessm.mayan_languages-screenshot3.jpg"
       ],
-      "es": [
-        "com.endlessm.mayan_languages-screenshot1.jpg",
-        "com.endlessm.mayan_languages-screenshot2.jpg",
-        "com.endlessm.mayan_languages-screenshot3.jpg"
-      ]
+      "es": []
```

Comment:
Pretty sure you can just delete this whole line now that there's nothing in here.